addons/survey/wizard/survey_invite.py

```diff
@@ -191,24 +165,29 @@ def _prepare_answers(self, partners, emails):
                 raise UserError(_('Some of recipients already started survey. Please update recipients list accordingly.'))
 
         for new_partner in partners - partners_done:
-            answers |= self.survey_id._create_answer(partner=new_partner, deadline=self.deadline)
+            answers |= self.survey_id._create_answer(partner=new_partner, **self._get_answers_values(partner=new_partner))
         for new_email in [email for email in emails if email not in emails_done]:
-            answers |= self.survey_id._create_answer(email=new_email, deadline=self.deadline)
+            answers |= self.survey_id._create_answer(email=new_email, **self._get_answers_values(email=new_email))
 
         return answers
 
+    def _get_answers_values(self, partner=False, email=False):
```

Comment:
why having some parameters and no using them ? O_0

Reply:
Probably remaining of WIP. Indeed removed.